static/src/javascripts/projects/common/modules/experiments/tests/adblocking-response.js

```diff
@@ -25,58 +25,58 @@ define([
-            ['user help email no-close'],
+            ['adblock supporter no-close'],
+            ['adblock subscriber no-close'],
             ['adblock whitelist immediate-close'],
-            ['adblock membership immediate-close'],
-            ['subscriber number immediate-close'],
-            ['user help email immediate-close'],
+            ['adblock supporter immediate-close'],
+            ['adblock subscriber immediate-close'],
             ['adblock whitelist delayed-close'],
-            ['adblock membership delayed-close'],
-            ['subscriber number delayed-close'],
-            ['user help email delayed-close']
+            ['adblock supporter delayed-close'],
+            ['adblock subscriber delayed-close'],
+            ['survey adblock delayed-dismissed'],
+            ['survey adblock immediate-dismissed']
         ].map(function(_) {
             return _[0];
         }).join(', ');
         this.dataLinkNames = 'adblock response overlay: '+variantDataLinkNames+', subscriber number page: user help email';
-        this.idealOutcome = 'Adblocking users will either pay or allow ads.';
+        this.idealOutcome = 'After whitelisting our ads, former ad-blocking users will not re-block them';
 
         this.canRun = function () {
-            return !contains('firefox', detect.getUserAgent.browser.toLowerCase())
+            return contains('chrome', detect.getUserAgent.browser.toLowerCase())
                 && contains(['desktop', 'leftCol', 'wide'], detect.getBreakpoint())
                 && config.page.edition === 'UK'
                 && !cookies.get('gu_abm_x');
         };
 
+        var isQualified = function () {
+            return !config.page.isFront &&
+                !config.page.shouldHideAdverts &&
+                !userFeatures.isPayingMember() &&
+                config.page.webTitle !== 'Subscriber number form' &&
+                config.page.webTitle !== 'How to disable your adblocker for theguardian.com' &&
+                storage.local.isStorageAvailable() &&
+                !storage.local.get('gu.subscriber') &&
+                storage.local.get('gu.alreadyVisited') > 5;
+        };
+
         this.variants = [{
+            id: 'control',
+            test: function(){}
+        }, {
             id: 'no-close',
             test: function () {
                 detect.adblockInUse.then(function (adblockUsed) {
-                    if (adblockUsed && !config.page.isFront &&
-                        !userFeatures.isPayingMember() &&
-                        config.page.webTitle !== 'Subscriber number form' &&
-                        config.page.webTitle !== 'How to disable your adblocker for theguardian.com' &&
-                        storage.local.isStorageAvailable() &&
-                        !storage.local.get('gu.subscriber')
-                    ) {
+                    if (adblockUsed && isQualified()) {
                         var surveyOverlay = new SurveyAdBlock({
```

Comment:
Couldn't we have only one instance of this object and use it in all `new SurveyAdBlock()` calls?